social/meeting-transcript/v0.1.0/schema.json

```diff
@@ -0,0 +1,135 @@
+{
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "$id": "https://common.schemas.verida.io/social/meeting-transcript/v0.1.0/schema.json",
+    "title": "Meeting Transcript",
+    "description": "Schema for a meeting transcript",
+    "type": "object",
+    "appearance": {
+        "style": {
+            "color": "#34A853",
+            "icon": "./icon.svg"
+        }
+    },
+    "database": {
+        "name": "social_meeting_transcript",
+        "indexes": {         
+            "name": ["name"],   
+            "inserted": ["insertedAt"],
+            "date": ["date"],            
+            "source": ["sourceApplication", "sourceId", "calendar_id"]
+        }
+    },
+    "layouts": {
+        "create": ["name", "organizer_email", "participants", "calendar_id"],
+        "view": ["name", "organizer_email", "participants", "date", "duration", "user", "sentence", "summary"]
+    },
+    "allOf": [
+        {
+            "$ref": "https://core.schemas.verida.io/base/v0.1.0/schema.json"
+        },
+        {
+            "type": "object",
+            "properties": {                
+                "organizer_email": {
```

Comment:
We use camelCase, so should be `organizerEmail`.

Make all other fields camelCase as well.